src/sentry/conf/server.py

```diff
@@ -518,6 +518,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
 CELERY_ACCEPT_CONTENT = {"pickle"}
 CELERY_IMPORTS = (
     "sentry.data_export.tasks",
+    "sentry.demo.tasks",
```

Comment:
Just realised, you can add this in the `demo/settings.py` file so that we don't import it in core sentry.

Something like
`CELERY_IMPORTS = CELERY_IMPORTS + ("sentry.demo.tasks",)